docs/creating_dataset.md

```diff
@@ -19,7 +19,11 @@ InnerEye expects segmentation datasets to have the following structure:
  at minimum:
     * `subject`: A unique positive integer assigned to every patient
     * `channel`: The imaging channel or ground truth structure described by this row. 
-    * `filePath`: Path to the Nifti file for this scan or structure
+    * `filePath`: Path to the file for this scan or structure. We support nifti (nii.gz), numpy (npz) and hdf5(h5).
+        * For HDF5 path suffix with | separator
+            * For images <path>|<dataset_name>|<channel index>
+            * For segmentation binary <path>|<dataset_name>|<channel index>
+            * For segmentation multimap <path>|<dataset_name>|<channel index>|<multimap value>
```

Comment:
We also need to add here what assumption we make about the ordering of the array. Which index in the HDF5 array corresponds to X, Y, Z? Are we dropping any dimension? 
Segmentation binary and multimap should be better described.